templates/ManageApprovers.mustache

```diff
@@ -29,19 +29,72 @@
 							</form>
 						</div>
 					{{/categories}}
-					<form method="POST" action="" class="add-category-form">
-						<input type="text" name="category" placeholder="New Category" required>
+					<form method="POST" class="add-category-form">
+						<input type="text" name="category" placeholder="New Category" id="{{username}}category" required>
```

Comment:
Somewhat unrelated question: what happens when username contains a "? Is it escaped?

Reply:
![Screenshot from 2024-08-17 23-42-09](https://github.com/user-attachments/assets/0bf84831-7597-40be-821a-07f11d6eea79)

Reply:
`"`, not `?`

Reply:
@JeroenDeDauw " and ' also work fine.

![Screenshot from 2024-08-18 12-52-30](https://github.com/user-attachments/assets/ecc4defc-6e20-47fe-a236-67ab4a330a42)